packages/server/lib/request.coffee

```diff
@@ -130,6 +130,12 @@ reduceCookieToArray = (c) ->
 createCookieString = (c) ->
   reduceCookieToArray(c).join("; ")
 
+propertiesToLowerCase = (obj) ->
```

Comment:
This mutates `options.headers`. We probably don't want to change the casing of every key in the object in light of #2879.

Perhaps just making a case-insensitive getter function would be better, something like `caseInsensitiveGet`?